Pat Mifsud/week01/homework/script.js

```diff
@@ -0,0 +1,168 @@
+
+// Calculator
+// PART 1
+
+const round2 = function(input){
+  return Math.round(input * 100) / 100;
+} //I wanted to round numbers to 2 decimal places at most, so made it a funciton
+
+const squareNumber = function(num, log){
+  const result = Math.pow(num, 2);
+  const statement =
+    `The result of squaring the number ${num} is ${round2(result)}.`
+  if (log){
+    console.log(statement);};
+  return(result);
+}
+
+const halfNumber = function(num){
+  const result = num / 2;
+  const statement = `The result of halving the number ${num} is ${round2(result)}.`;
+  console.log(statement);
+  return(result);
+}
+
+const percentOf = function(num1, num2){
+  const result = (num1 / num2) * 100;
+  const statement = `${num1} is ${round2(result)}% of ${num2}.`
+  console.log(statement);
+  return(result);
+}
+
+const areaOfCircle = function(radius){
+  const result = squareNumber(Math.PI * radius, false);
+  const statement =
+    `The area for a circle with radius ${radius} is ${round2(result)}.`
+  console.log(statement);
+  return(result);
+
+}
+
+// PART 2
+
+const partTwoFunction = function(num=42){
+  const half = halfNumber(num);
+  const squa = squareNumber(half);
+  const circ = areaOfCircle(squa);
+  const perc = percentOf(circ, squa);
+  const result = round2(perc);
+  const statement = `The half/squared/ared/percented result is ${result}`;
+  console.log(statement);
+  return(round2(result));
+}
+
+// Part 1 print
+
+console.log("");
+console.log("📚 PART 1");
+squareNumber(44, true);
+halfNumber(3244);
+percentOf(40, 200);
+areaOfCircle(342);
+console.log("");
+
+console.log(`📚 PART 2`);
+partTwoFunction();
+console.log("");
+
+
+
+// Strings
+//
+const drEvil = function(amount){
+  let pinky = "."
+  if (amount === 1000000){
+    pinky = "(pinky)."}
```

Comment:
```
if (amount === 1000000) {
     pinky = "(pinky).";
}
```
I will recommend you to put the ending `}` in a separate line and end the previous line of code with a `;`